public/app/ui/pages/static/home.html

```diff
@@ -1,20 +1,37 @@
 <article id="home">
     <h1>Research Western Australian History</h1>
     <p>
-    Access to the Royal Western Australian Historical Society's collection for research and public interest purposes.
+        This website provides access to the Royal Western Australian Historical Society's collection for research and public interest purposes. Searches can be performed on the <a href="/photographs/search">photographic</a> and <a href="/museum/search">museum</a> collections, as well as <a href="/memorials/search">memorial records</a> and the <a href="/library/search">library catalogue</a>.
     </p>
-    <div class="row">
-        <div class="col-md-3 col-sm-6 col-xs-12 text-center">
-            <a href="/library/search" class="thumbnail">Library Search</a>
+    <div class="row homepage">
+        <div class="col-md-4 col-sm-12 col-xs-12 text-center">
+            <div class="well">
+                <h2><a href="/photographs/search">Search the photographic collection</a></h2>
+                <img src="images/photo-search-feature.jpg" alt="Launch of pilot boat, Paddy Troy, on May Day 1985" />
+                <p><a href="/photographs/search" class="btn btn-primary active" role="button">Access digitised photography</a></p>
+            </div>
         </div>
-        <div class="col-md-3 col-sm-6 col-xs-12 text-center">
-            <a href="/photographs/search" class="thumbnail">Photographs Search</a>
-        </div>
-        <div class="col-md-3 col-sm-6 col-xs-12 text-center">
-            <a href="/museum/search" class="thumbnail">Museum Search</a>
-        </div>
-        <div class="col-md-3 col-sm-6 col-xs-12 text-center">
-            <a href="/memorials/search" class="thumbnail">Public Memorials Search</a>
+        <div class="col-md-8 col-sm-12 col-xs-12 text-center">
+            <div class="well">
+                <h2><a href="/library/search">Library Search</a></h2>
```

Comment:
Not having an image for Library is possibly too inconsistent.  Especially in mobile view, it is much less prominent than the other three collections.